api/src/main/java/org/openmrs/module/locationbasedaccess/utils/LocationUtils.java

```diff
@@ -61,4 +61,16 @@ public static Boolean doesUsersForPersonBelongToGivenLocation(Person person, Str
     public static Boolean compare(String value1, String value2) {
         return (StringUtils.isNotBlank(value1) && StringUtils.isNotBlank(value2)) && value1.equals(value2);
     }
+
+    public static String getUserAccessibleLocationUuid(User authenticatedUser) {
```

Comment:
Can you add some javadoc to explain what this method is to be used for?

Reply:
Added